src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```diff
@@ -254,6 +254,48 @@
         }
       ]
     },
+    "XUnitVersion": {
+      "type": "parameter",
+      "description": "The version of xUnit.net to use.",
+      "displayName": "xUnit.net version",
+      "datatype": "choice",
+      "isEnabled": "TestFramework == xUnit.net",
+      "replaces": "XUnitVersion",
+      "choices": [
+        {
+          "choice": "v2",
+          "displayName": "v2",
+          "description": "Chooses xUnit v2 with VSTest"
+        },
+        {
+          "choice": "v3",
+          "displayName": "v3",
+          "description": "Chooses xUnit v3 with VSTest (Aspire version 9.1 and later only)"
+        },
+        {
+          "choice": "v3mtp",
+          "displayName": "v3 with MTP",
+          "description": "Chooses xUnit v3 with Microsoft Test Platform (Aspire version 9.1 and later only)"
+        }
+      ],
+      "defaultValue": "v2"
+    },
+    "TestFrameworkIsXUnit": {
+      "type": "generated",
+      "generator": "switch",
+      "parameters": {
+        "cases": [
+          {
+            "condition": "(TestFramework == xUnit.net)",
```

Comment:
this doesn't seem to be working
![image](https://github.com/user-attachments/assets/533d5c34-45b9-4310-9524-bb5fa7972069)